NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -143,6 +148,11 @@ static const tConfigurationEntries defaultConfiguration =
     { "*UsoIPv4", 1, 0, 1},
     { "*UsoIPv6", 1, 0, 1},
 #endif
+#if NDIS_SUPPORT_NDIS650
+    #if ALICLOUD_MAX_FRAGMENTS_IN_ONE_NB
+        {"*MaxFragments",ALICLOUD_MAX_FRAGMENTS_IN_ONE_NB,1,1024},
```

Comment:
Please don't use "*" for the private parameters. "*" reserved for Microsoft defined parameters.

Reply:
Okay, I will adjust it to a different parameter name, AliCloudMaxFragments. How about that?

Reply:
@ybendito, 
1. ok. how about MaxFragmentsInOneNetBuffer?    
2. Regarding the parameter upper and lower limits, I am not sure about the range of other vendors and whether 16-256 is sufficient. (For Alicloud, the current support capability is already up to 256.) Considering that this parameter is stored in the registry and is not present by default, would it be better to have a larger range in my understanding? I also noticed that the range given for the parameter NumberOfHandledRXPacketsInDPC is quite extensive.

Reply:
@zjmletang Yes, Something like MAX_FRAGMENTS_IN_ONE_NB looks much better
High value of NumberOfHandledRXPacketsInDPC does not affect anything but too high number of max fragments is just wrong, with indirect it can't be bigger than capacity of indirect SG buffer, without indirect it can't be bigger than TX size.

Reply:
In the case where TSO is enabled (i.e. actually always) the max size of the packet is close to 64K. If this is the regular app that transmits big data chunk the typical number of fragments is ~18. In lights of **https://github.com/virtio-win/kvm-guest-drivers-windows/issues/972** this is the target case also for VM setup. What we need is the steps to reproduce it. I will make a note about it in the issue.

Reply:
@ybendito , I commented on the reproduction steps in comment #972.
If continuous TCP short packets are sent, it is not the case that the entire 4KB page is used to store the packet; one page might only contain a single short packet. When the Nagle's algorithm combines these successive TCP short packets, the number of pages ultimately used could be quite high. This scenario of continuously sending small packets is used by some clients in the gaming industry.